workflows/microbiome/gene-based-pathogen-identification/Gene-based-Pathogen-Identification.ga

```diff
@@ -0,0 +1,1165 @@
+    },
+    "steps": {
+        "0": {
+            "annotation": "Nanopore Sequenced Reads Collection",
+            "content_id": null,
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Nanopore Sequenced Reads Collection",
+                    "name": "nanopore_sequenced_reads_collection"
+                }
+            ],
+            "label": "nanopore_sequenced_reads_collection",
+            "name": "Input dataset collection",
+            "outputs": [],
+            "position": {
+                "left": 0,
+                "top": 270.7499885559082
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null, \"collection_type\": \"list\"}",
+            "tool_version": null,
+            "type": "data_collection_input",
+            "uuid": "68bf45e2-7aa6-47b1-af6e-c3f8e2802e2a",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "1": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/collection_element_identifiers/collection_element_identifiers/0.0.2",
+            "errors": null,
+            "id": 1,
+            "input_connections": {
+                "input_collection": {
+                    "id": 0,
+                    "output_name": "output"
+                }
+            },
+            "inputs": [],
+            "label": null,
+            "name": "Extract element identifiers",
+            "outputs": [
+                {
+                    "name": "output",
+                    "type": "txt"
+                }
+            ],
+            "position": {
+                "left": 300,
+                "top": 19.749984741210938
+            },
+            "post_job_actions": {
+                "RenameDatasetActionoutput": {
+                    "action_arguments": {
+                        "newname": "extracted_lables"
+                    },
+                    "action_type": "RenameDatasetAction",
+                    "output_name": "output"
+                }
+            },
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/collection_element_identifiers/collection_element_identifiers/0.0.2",
+            "tool_shed_repository": {
+                "changeset_revision": "d3c07d270a50",
+                "name": "collection_element_identifiers",
+                "owner": "iuc",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"input_collection\": {\"__class__\": \"ConnectedValue\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "0.0.2",
+            "type": "tool",
+            "uuid": "11aa5d81-81ba-4c37-89cd-5774c41d3e63",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "label": "extracted_lables",
```

Comment:
What does "extracted_lables" mean?

Reply:
Sample IDs

Reply:
Can you have a name or label for the outputs that clearly say what are they?
Here there is a spelling mistake I think also